### PR TITLE
feat: improve profitable harvest test

### PR DESCRIPTION
**Pull request by TheRyanMiller**
Improved tests for `test_profitable_harvest`.

- Add a check to ensure pricePerShare has increased
- Include vault balance for profit check rather than the strategy
- Add sender for harvest txns (makes it easier to test against live contracts)

### CODE REVIEW

**Comment by TheRyanMiller** [comment]
I’m really bad a git 😅. Please ignore for now. Gonna close this and reopen one that passes tests when I get a chance.

**Comment by fubuloubu** [comment]
> I’m really bad a git . Please ignore for now. Gonna close this and reopen one that passes tests when I get a chance.

I think you're doing okay! p.s. not sure if you've see these for commit messages: https://www.conventionalcommits.org/en/v1.0.0/

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "@commitlint/config-conventional": "^11.0.0",
     "ethlint": "^1.2.5",
     "husky": "^4.3.0",
-    "prettier": "^2.1.2",
+    "prettier": "^2.2.1",
```
Comment (fubuloubu) [inline package.json:8]: Can this... be in another PR? The change to the lock file is big

Comment (TheRyanMiller) [reply package.json:8]: this came automatically i think as a result of `npm i prettier` when trying to troubleshoot.
